asus-ec-sensors.c

```diff
@@ -477,6 +477,15 @@ static const struct ec_board_info board_info_strix_b550_i_gaming = {
 	.family = family_amd_500_series,
 };
 
+static const struct ec_board_info board_info_strix_b650e_i_gaming = {
+	.sensors = SENSOR_SET_TEMP_CHIPSET_CPU_MB |
+		SENSOR_TEMP_T_SENSOR | SENSOR_TEMP_VRM |
+		SENSOR_FAN_VRM_HS | SENSOR_CURR_CPU |
```

Comment:
Why don't we see all the sensors in the provided `sensors` output?

Reply:
because the board doesn't have some of these sensors like a VRM fan AFAIK 😅. I started by copy/pasting the sensors list from the b550-i board as a reference since i have that board too and I got sloppy and didn't remove the extras. Sorry about that, pushing up a commit with _just_ the sensors i could get working